src/app/components/character-card/character-card.component.html

```diff
@@ -0,0 +1,12 @@
+<div [ngClass]="isCharacterSelected ? 'card selected' : 'card'">
+  <img class="picture" [src]="character.image" [alt]="character.name">
+  <h2 class="title">{{ character.name }}</h2>
+  <span class="tag" [ngClass]="statusClass">{{ character.status }}</span>
+  <p class="detail">
+    <strong class="bold">Gender:</strong> {{ character.gender }}
```

Comment:
Bold class is necessary?

Reply:
Yeah, the ```strong``` tag is not making any effect, perhaps is for the equalizer file.